### PR TITLE
consensus: log private validator address and not struct

### PR DESCRIPTION
**Pull request by alexanderbez**
ref: https://github.com/tendermint/tendermint/pull/6140#discussion_r578635756

### CODE REVIEW

File: consensus/state.go [hunk 1]
```diff
@@ -1095,15 +1095,15 @@ func (cs *State) enterPropose(height int64, round int32) {
 		logger.Debug(
 			"propose step; our turn to propose",
 			"proposer", address,
-			"priv_validator", cs.privValidator,
+			"priv_validator", address,
```
Comment (marbar3778) [inline consensus/state.go:1098]:
```suggestion
```
address is above

File: consensus/state.go [hunk 2]
```diff
@@ -1095,15 +1095,15 @@ func (cs *State) enterPropose(height int64, round int32) {
...
 		)
 
 		cs.decideProposal(height, round)
 	} else {
 		logger.Debug(
 			"propose step; not our turn to propose",
 			"proposer", cs.Validators.GetProposer().Address,
-			"priv_validator", cs.privValidator,
+			"priv_validator", address,
```
Comment (marbar3778) [inline consensus/state.go:1106]:
```suggestion
```
To keep consist with what's on 0.34

File: consensus/state.go (hunk 1 above)
Comment (alexanderbez) [reply consensus/state.go:1098]: Yup.

**Review by marbar3778 — APPROVED** [review]

**Review by cmwaters — APPROVED** [review]

**Comment by tessr** [comment]
Thank you for doing this! It got lost in the Wednesday night shuffle.

**Pull request merged**